src/projects/shared/site/src/containers/Header/Right/SettingsMenu/index.tsx

```diff
@@ -30,7 +30,7 @@ export const SettingsMenu = () => {
     //  especially the gross 'useAPIMethods' thing
     useEffect(() => {
         designer.viewport.debugOptions = debugInfo;
-    }, [designer, ...Object.values(debugInfo)]); // Updates when any of the debugInfo values change
+    }, [debugInfo, designer]); // Updates when any of the debugInfo values change
```

Comment:
does this work though? these dependency lints actually changed behavior so I'm not as keen on letting them through like that